src/dump.js

```diff
@@ -62,6 +62,20 @@ QUnit.dump = ( function() {
 			},
 			typeOf: function( obj ) {
 				var type;
+
+				function isArray() {
```

Comment:
This does make sense as its own function, but it shouldn't be defined inside `typeOf`... can you give it a parameter and move it up to line 37?